airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1425,6 +1425,8 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/ConnectionCollectionItem'
+        total_entries:
+          type: integer
```

Comment:
While this will work, we have a CollectionInfo component. I will suggest we use it and expand to other Collection components that do not have total_entries.
For the ConnectionCollection, I would suggest this:
```
ConnectionCollection:
      type: object
      description: Connections
      allOf:
       - type: object
         properties:
           connections:
            type: array
            items:
              $ref: '#/components/schemas/ConnectionCollectionItem'
       - $ref: '#/components/schemas/CollectionInfo'
  ```
This is because the CollectionInfo component explains what total_entries is and since this change will affect many components, it's better we use the CollectionInfo component. 
You can expand this change to DAGRun, Pool, Variable etc

Reply:
`allOf` is always a headache in openapi 3.0, but surprisingly this one works 
my guess is because `CollectionInfo` is not nullable otherwise we are [doomed](https://github.com/OAI/OpenAPI-Specification/issues/1368) :joy: 
i'll add the rest, thanks :+1:

Reply:
@ephraimbuddy done :rocket: 
i updated all collections for all available API endpoints except `ExtraLinkCollection` and `TaskCollection` which don't have `total_entries`

Reply:
> > i had to fix a few other  to make these endpoints work in my python sdk, for example, several missing `nullable` items and some missing properties like `running_slots` in `Pool`
> > I'll create new PRs for them
> 
> Let's fix only the collection info for this PR, You can make another PR for another one.
> And since you have added collection info to the components, you have to remove it from the 200 responses of the affected endpoints where we also have collection info right now. E.g :
> https://github.com/apache/airflow/blob/11d03d2f63d88a284d6aaded5f9ab6642a60561b/airflow/api_connexion/openapi/v1.yaml#L270-L271
> 
> You have to remove CollectionInfo here since it's already part of the other component

ha, i actually removed them long ago since it never worked with the openapi python generator
but someone here told me not to update it since other generator may need it